src/main/java/frc/robot/subsystems/swerve/modules/Module.java

```diff
@@ -157,6 +158,14 @@ public Rotation2d getCurrentAngle() {
         return moduleInputsContainer.getSteerMotorInputs().angle;
     }
 
+    public String getModuleLogPath() {
+        return constants.logPath() + modulePosition + "/";
```

Comment:
save this logpath as תכונה or get it in module constants instead of getting logprefix and modulepostion separately